src/app/_components/DistrictSearch/List.js

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { Tick } from '@icons/index';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { SwiperSlide } from 'swiper/react';
+import { PillButton, Slider } from '@components';
```

Comment:
🟠 by convention it should be splited into direct imports

Reply:
Could you please explain?

Reply:
```
import { PillButton } from '@components/PillButton';
import { Slider } from '@components/Slider';
```

Reply:
I might be missing something. If we go that way using directed imports, then why do we have an index.js in the components folder?

Reply:
Fixed it